tests/python/frontend/nnef/cases/le_4d_broadcast/graph.nnef

```diff
@@ -0,0 +1,8 @@
+version 1.0;
+
+graph G( input1, input2 ) -> ( output )
+{
+    input1 = external<scalar>(shape = [4,16,32,32]);
+    input2 = external<scalar>(shape = [1,16,1,1]);
```

Comment:
given this is text format, might be useful to directly embed in python

Reply:
Fixed in https://github.com/apache/tvm/pull/17053/commits/4b3c838984f26acf61bcdb3c8da209fbe49446d6